alws/crud/build_node.py

```diff
@@ -162,6 +213,29 @@ async def __process_logs(pulp_client: PulpClient, task_id: int,
                     href=href
                 )
             )
+    if exceptions:
+        raise ArtifactConversionError(
+            'Cannot put build logs into Pulp storage: %s',
+            '\n'.join([str(e) for e in exceptions]))
+
+    for repo_href, files in repo_files_mapping.items():
+        try:
+            await pulp_client.modify_repository(repo_href, add=files)
+        except Exception as e:
+            logging.error('Cannot add log files to the repository: %s',
+                          str(e))
+            exceptions.append(e)
+            continue
+        try:
+            await pulp_client.create_file_publication(repo_href)
+        except Exception as e:
+            logging.error('Cannot create publication for log files: %s',
+                          str(e))
+            exceptions.append(e)
```

Comment:
Adding logs in repo, seems like logic for adding rpms/modules in repos, can we use it like seperate func?

Reply:
Yes, no, maybe... They use different error messages, but I'll try to move the logic into separate function

Reply:
It's up to you anyway)